catenets/models/constants.py

```diff
@@ -18,8 +20,8 @@
 DEFAULT_STEP_SIZE = 0.0001
 DEFAULT_STEP_SIZE_T = 0.0001
 DEFAULT_N_ITER = 10000
-DEFAULT_BATCH_SIZE = 100
-DEFAULT_PENALTY_L2 = 0.01 / 100
+DEFAULT_BATCH_SIZE = 512

```

Comment:
Can we keep the original default batch size of 100? Not only did 100 work well for me and has created all results in my papers, but it was also what was used in e.g. the original TARNet code.